index.html

```diff
@@ -1956,7 +1956,8 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
                 with the given URI or IRI.</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a></td></tr>
 <tr class="rfc2119-table-assertion" id="td-vocab-sizes--Link"><td><code>sizes</code></td><td>Target attribute that specifies one or more sizes for the referenced icon. 
                     Only applicable for relation type "icon". The value pattern follows 
-                    {Height}x{Width} (e.g., "16x16", "16x16 32x32").</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr></tbody></table> <p>Link relations can be used to describe relations such as to other Things (e.g., a Switch Thing controls a Lamp Thing), to a specific kind of Thing Models (e.g., a Thing Description
+                    {Height}x{Width} (e.g., "16x16", "16x16 32x32").</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-hreflang--Link"><td><code>hreflang</code></td><td>The hreflang attribute specifies the language for a linked document. The value of this attribute should conform to [[RFC5646]].</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr></tbody></table> <p>Link relations can be used to describe relations such as to other Things (e.g., a Switch Thing controls a Lamp Thing), to a specific kind of Thing Models (e.g., a Thing Description
```

Comment:
BCP47 is a better reference than RFC5646, since it is durable (5646 replaced 4646 replaced 3066 replaced 1766...)

"should conform to" is a little obscure? I would probably say "should be a valid language tag [[BCP47]]". Note that the term "valid" has a specific meaning in BCP47.